refresh/templates/fromswagger/Routes.swift.hbs

```diff
@@ -17,14 +17,143 @@ import Kitura
-{{#each routes}}
+{{#if generatecodable}}
+  {{#if hascodable}}
+    // Handlers for codable routes are declared below:
+    //
+  {{/if}}
+  {{#each routedetail}}
+    {{#ifequal this.method "delete"}}
+      {{#if this.codable}}
+        {{#ifequal this.variant "all"}}
+    func deleteAll_{{this.handlerName}}(completion: (RequestError?) -> Void ) -> Void {
+        completion(.notImplemented)
+    }
+        {{/ifequal}}
+        {{#ifequal this.variant "id"}}
+    func deleteOne_{{this.handlerName}}(id: {{this.idtype}}, completion: (RequestError?) -> Void ) -> Void {
+        completion(.notImplemented)
+    }
+
+        {{/ifequal}}
+      {{/if}}
+    {{/ifequal}}
+    {{#ifequal this.method "get"}}
+      {{#if this.codable}}
+        {{#ifequal this.variant "all"}}
+    func getAll_{{this.handlerName}}(completion: ([{{this.response}}]?, RequestError?) -> Void ) -> Void {
+        completion(nil, .notImplemented)
+    }
+        {{/ifequal}}
+        {{#ifequal this.variant "id"}}
+    func getOne_{{this.handlerName}}(id: {{this.idtype}}, completion: ({{this.response}}?, RequestError?) -> Void ) -> Void {
+        completion(nil, .notImplemented)
+    }
+        {{/ifequal}}
+        {{#ifequal this.variant "noid"}}
+    func get_{{this.handlerName}}(completion: ({{this.response}}?, RequestError?) -> Void ) -> Void {
+        completion(nil, .notImplemented)
+    }
+        {{/ifequal}}
+
+      {{/if}}
+    {{/ifequal}}
+    {{#ifequal this.method "patch"}}
+      {{#if this.codable}}
+    func patchOne_{{this.handlerName}}(id: {{this.idtype}}, patch: {{this.param}}, completion: ({{this.param}}?, RequestError?) -> Void ) -> Void {
+        completion(nil, .notImplemented)
+    }
+
+      {{/if}}
+    {{/ifequal}}
+    {{#ifequal this.method "post"}}
+      {{#if this.codable}}
+        {{#ifequal this.variant "noid"}}
+    func post_{{this.handlerName}}(post: {{this.param}}, completion: ({{this.param}}?, RequestError?) -> Void ) -> Void {
+        completion(nil, .notImplemented)
+    }
+        {{/ifequal}}
+        {{#ifequal this.variant "id"}}
+    func postOne_{{this.handlerName}}(post: {{this.param}}, completion: ({{this.idtype}}?, {{this.param}}?, RequestError?) -> Void ) -> Void {
+        completion(nil, nil, .notImplemented)
+    }
+        {{/ifequal}}
+
+      {{/if}}
+    {{/ifequal}}
+    {{#ifequal this.method "put"}}
+      {{#if this.codable}}
+    func putOne_{{this.handlerName}}(id: {{this.idtype}}, put: {{this.param}}, completion: ({{this.param}}?, RequestError?) -> Void ) -> Void {
+        completion(nil, .notImplemented)
+    }
+
+      {{/if}}
+    {{/ifequal}}
+  {{/each}}
+
+  {{#if hascodable}}
+    // Codable routes are declared below:
+    //
+  {{/if}}
+  {{#each routedetail}}
+{{#if this.codable}}
+    {{#ifequal this.method "delete"}}
+      {{#ifequal this.variant "all"}}
+    app.router.{{this.method}}("{{#if ../basepath}}\(basePath){{/if}}{{this.route}}", handler: deleteAll_{{this.handlerName}})
```

Comment:
I believe we should verify the BasePath before

Reply:
Enrique and I agree this should not be changed